src/api/organizations.ts

```diff
@@ -7,8 +7,7 @@ export type Organization = {
 }
 
 export const fetchOrganizations = async (token: string): Promise<Organization[]> => {
-    const url = '/v1/organizations'
-    const response = await apiClient.get(url, {
+    const response = await axiosClient.get('/v1/organizations', { // TODO why does swagger not have /v1/organizations?
```

Comment:
I think it's because the `organizations` API is part of the internal docs, is there something we can do there?

Reply:
yeah i'm thinking they should just be in the pubic docs anyway since they're public endpoints? i created a ticket to figure it out here https://taplytics.atlassian.net/browse/DVC-7723